books/management/commands/addepub.py

```diff
@@ -195,11 +195,13 @@ def process_epub(self, filename, use_symlink=False):
                             auth = fix_authors(auth)
                             if auth:
                                 for a in auth if not \
-                                        isinstance(auth, basestring) else [auth]:
+                                        isinstance(auth, basestring) \
+                                        else [auth]:
```

Comment:
This was a bit cumbersome - if `fix_authors` is not used anywhere else, perhaps it would be a good idea to modify the function so it *always* returns a list (even if it only has one element), maybe even including the strip and replace stuff at 192-193, for clarity.